sources/packages/web/src/components/layouts/Institution/InstitutionFullLayout.vue

```diff
@@ -8,7 +8,7 @@
     container with main, when vuetify3 stable
       version is relased -->
     <v-container fluid>
-      <router-view></router-view>
+      <router-view :key="$route.fullPath"></router-view>
```

Comment:
why do we want a "force a replacement"

Reply:
I changed to reload the data watching the location id and it resulted in a better UX.
Thanks for raising the question 👍